test/functional/unified-spec-runner/operations.ts

```diff
@@ -42,8 +43,11 @@ operations.set('aggregate', async ({ entities, operation }) => {
       hint: operation.arguments.hint,
       let: operation.arguments.let,
       out: operation.arguments.out
-    })
-    .toArray();
+    });
+  if (session) {
```

Comment:
If a unified test specifies a named session in the `arguments`, we need to set it on the cursor.